common/log.go

```diff
@@ -156,3 +158,9 @@ func setupFileHook(hook config.LogrusHook, level logrus.Level, componentName str
 		),
 	})
 }
+
+func LogIfError(ctx context.Context, err error, message string) {
+	if err != nil {
+		util.GetLogger(ctx).WithError(err).Error(message)
```

Comment:
`util.GetLogger` will not accept a `nil` Context. We don't happen to pass in a nil context anywhere I can see, but it might be nice to sanity check with something like:
```
if ctx == nil {
    ctx = context.TODO()
}
```
which will save people a potential headache.